docker-compose.yml

```diff
@@ -252,6 +252,7 @@ services:
     volumes: *volumes
     environment:
       - DATABASE_URL=postgres://origin:origin@postgres/origin
+      - REDIS_URL=redis://redis-master:6379
```

Comment:
Need a DB number here?  e.g. `redis://redis-master:6379/0`  If it defaults to 0, cool.

Reply:
Looks like it doesn't need it - see the other REDIS_URL env in the docker-compose.yml